src/wp-includes/customize/class-wp-customize-nav-menus-panel.php

```diff
@@ -98,4 +98,25 @@ protected function content_template() {
 		<li class="customize-control-title customize-section-title-nav_menus-heading"><?php _e( 'Menus' ); ?></li>
 		<?php
 	}
+
+	/**
+	 * Checks required user capabilities and whether the theme has the
+	 * feature support required by the panel.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @return bool False if theme doesn't support the panel or the user doesn't have the capability.
+	 */
+	public function check_capabilities() {
+		// WP_Customize_Panel::$theme_supports only supports checking one
+		// theme_supports, so instead we override check_capabilities().
```

Comment:
```suggestion
		/* 
		 * WP_Customize_Panel::$theme_supports only supports checking one
		 * theme_supports, so instead we override check_capabilities().
		 */
```

CS Nit, multi line comments.